src/components/UserSelector.tsx

```diff
@@ -22,13 +33,21 @@ export const UserSelector: React.FC = () => {
       </div>
 
       <div className="dropdown-menu" id="dropdown-menu" role="menu">
-        <div className="dropdown-content">
-          <a href="#user-1" className="dropdown-item">Leanne Graham</a>
-          <a href="#user-2" className="dropdown-item is-active">Ervin Howell</a>
-          <a href="#user-3" className="dropdown-item">Clementine Bauch</a>
-          <a href="#user-4" className="dropdown-item">Patricia Lebsack</a>
-          <a href="#user-5" className="dropdown-item">Chelsey Dietrich</a>
-        </div>
+        {dropdownSelector && (
+          <div className="dropdown-content">
+            {users.map(user => (
+              <a
+                key={user.id}
+                href={`#user-${user.id}`}
+                className="dropdown-item"
+                onClick={() => handleSelectUser(user)}
+              >
+                {user.name}
+              </a>
+            ))}
+          </div>
+        )}
+        {/* <a href="#user-2" className="dropdown-item is-active">Ervin Howell</a> */}
```

Comment:
![image](https://github.com/mate-academy/react_dynamic-list-of-posts/assets/55200530/3fa65566-0a7f-43e2-9f84-f3f92c955f0f)